pyanaconda/ui/gui/spokes/installation_source.glade

```diff
@@ -91,14 +91,14 @@
             <property name="can_focus">False</property>
             <property name="layout_style">end</property>
             <child>
-              <object class="GtkButton" id="doneButton">
-                <property name="label" translatable="yes" context="GUI|Software Source|Media Check Dialog">_Done</property>
+              <object class="GtkButton" id="closeButton">
+                <property name="label" translatable="yes" context="GUI|Software Source|Media Check Dialog">_Close</property>
```

Comment:
Shouldn't it be called `Cancel`?

Reply:
`Cancel` is fine as well and makes more sense when the verification is in progress - changed.